common/src/test/java/org/conscrypt/MacTest.java

```diff
@@ -163,6 +166,10 @@ public void test(final Provider provider, final String algorithm) throws Excepti
     @Test
     public void invalidKeyThrows() {
         ServiceTester.test("Mac")
+            // BC actually accepts RSA public keys for these algorithms. No other Android
```

Comment:
Even though no other provider provides these, can we explicitly target BC with these exclusions, to be future proof?